protocol-units/execution/util/src/config/common.rs

```diff
@@ -120,6 +120,22 @@ env_default!(
 	10
 );
 
+env_default!(default_maptos_ledger_prune_window, "MAPTOS_LEDGER_PRUNING_WINDOW", u64, 50_000_000);
+
+env_default!(
+	default_maptos_state_merkle_prune_window,
+	"MAPTOS_STATE_MERKLE_PRUNING_WINDOW",
+	u64,
+	100_000 // see comment in aptos-core, this essentially needs to exceed TPS
+);
+
+env_default!(
+	default_maptos_epoch_snapshot_prune_window,
+	"MAPTOS_EPOCH_SNAPSHOT_PRUNING_WINDOW",
+	u64,
+	50_000_000
```

Comment:
What does this number represent? transactions? I need a little more context.

Reply:
Yeah, `ledger_version`. It's easiest to read the upstream comments if you want me to permalink them.

Reply:
e2e testing is...

```
just suzuka-full-node native build.setup.eth-local.celestia-local.test
```

Reply:
ah its the normal just e2e test, I wondered if some other test was added. Ok thanks.